Project.toml

```diff
@@ -5,21 +5,25 @@ version = "0.5.13"
 
 [deps]
 Arrow = "69666777-d1a9-59fb-9406-91d4454c9d45"
+ConstructionBase = "187b0558-2788-49d3-abe0-74a17ed4e7c9"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 
 [compat]
 Arrow = "2"
 Compat = "3.34, 4"
+ConstructionBase = "1.5"
 DataFrames = "1"
 Tables = "1.4"
 julia = "1.6"
 
 [extras]
+Accessors = "7d9f7c33-5ae7-4f3b-8dc6-eff91059b697"
+Aqua = "4c88cf16-eb10-579e-8560-4a9242c79595"
```

Comment:
Hm, I'm not sure.  Accessors in particular is still on 0.1, and I'm wary of adding additional compat hell just for tests...on the flip side, we wouldn't want our tests to break just because Accessors made a breaking release so...

Reply:
Probably mostly beneficial to have the compat bounds. If we end up upper-bounding the required test packages we avoid accidentally breaking tests and additionally continue testing an older version retains compatibility with the rest of our dependencies